lib/vimgolf/cli.rb

```diff
@@ -2,7 +2,6 @@ module VimGolf
 
   GOLFDEBUG    = ENV['GOLFDEBUG'].to_sym rescue false
   GOLFHOST     = ENV['GOLFHOST']     || "http://www.vimgolf.com"
-  GOLFDIFF     = ENV['GOLFDIFF']     || 'diff'
```

Comment:
Hmm, what about gvim users? If memory serves we had a couple of questions about that before and that's why the option was introduced.. to allow custom diff viewers.